data/mods/MindOverMatter/modinfo.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
+    "id": "mindovermatter",
+    "name": "Mind Over Matter",
+    "authors": [ "StandingStorm" ],
+    "maintainers": [ "StandingStorm" ],
+    "description": "Psionic powers for Cataclysm",
+    "category": "content",
+    "dependencies": [ "dda" ]
+  }
+]

```

Comment:
This one and the one below seem to be a sticking point for the tests and I'm not sure how to solve it. Committing the suggestion doesn't work because I get 

> The suggestion is invalid because no changes were made to the code

And trying to manually reformat it doesn't change anything either.